### PR TITLE
Standardize LICENSE body text formatting

### PR DESCRIPTION
**Pull request by stuartmorgan**
## Description

This cleans up a number of unnecessary differences in the LICENSE files:
- Removes C++ comments ("// ") from the beginnings of all the lines in license files that had them.
- Standardizes on having a blank line after the copyright line.
- Standardizes on having the final newline in the file.
- Standardizes on the indentation of the bulleted conditions.
- Standardizes on the locations of line breaks (re-wrapping the text)

This makes the license bodies--not including the copyright line--in almost all files the same, and matches the exact formatting already used in flutter/flutter and flutter/engine.

The only remaining non-copyright-line differences that I'm aware of at this point are:
- packages/image_picker/image_picker/LICENSE has an extra license
- packages/google_sign_in/extension_google_sign_in_as_googleapis_auth/LICENSE use "Google LLC" rather than "Google Inc." in the third condition.

## Related Issues

None

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: packages/image_picker/image_picker/LICENSE [hunk 1]
```diff
@@ -15,23 +15,22 @@ met:
...
+ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 --------------------------------------------------------------------------------
 aFileChooser
 
                                  Apache License
                            Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+                        http:www.apache.org/licenses/
```
Comment (amirh) [inline packages/image_picker/image_picker/LICENSE:33]: nit: keep the '//'

File: packages/image_picker/image_picker/LICENSE [hunk 2]
```diff
@@ -223,10 +222,10 @@ aFileChooser
    you may not use this file except in compliance with the License.
    You may obtain a copy of the License at
 
-       http://www.apache.org/licenses/LICENSE-2.0
+       http:www.apache.org/licenses/LICENSE-2.0
```
Comment (amirh) [inline packages/image_picker/image_picker/LICENSE:225]: ditto

**Review by amirh — APPROVED** [review]
LGTM modulo 2 comments inline

File: packages/image_picker/image_picker/LICENSE (hunk 1 above)
Comment (stuartmorgan) [reply packages/image_picker/image_picker/LICENSE]: Doh! I thought I put a `^` in that search/replace, but apparently not. Thanks for catching that!

File: packages/image_picker/image_picker/LICENSE (hunk 2 above)
Comment (stuartmorgan) [reply packages/image_picker/image_picker/LICENSE]: Done.

**Review by Hixie — APPROVED** [review]

**Pull request merged**